File: Assets/Standard Assets/Scripts/Camera Scripts/SmoothLookAt.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class SmoothLookAt : MonoBehaviour
{
    public Vector3 target = new Vector3();
    public float damping = 6.0f;
    public bool smooth = true;

    void LateUpdate()
    {
        // Look at and dampen the rotation
        var rotation = Quaternion.LookRotation(target - transform.position);
        transform.rotation = Quaternion.Slerp(transform.rotation, rotation, Time.deltaTime * damping);
    }

}

File: Assets/Standard Assets/Scripts/Camera Scripts/SmoothLookAt.js
var target : Vector3;
var damping = 6.0;
var smooth = true;

function LateUpdate () {
			// Look at and dampen the rotation
			var rotation = Quaternion.LookRotation(target - transform.position);
			transform.rotation = Quaternion.Slerp(transform.rotation, rotation, Time.deltaTime * damping);
}
